src/game_interpreter_map.cpp

```diff
@@ -655,10 +656,11 @@ bool Game_Interpreter_Map::CommandShowBattleAnimation(RPG::EventCommand const& c
 
 bool Game_Interpreter_Map::CommandFlashSprite(RPG::EventCommand const& com) { // code 11320
 	int event_id = com.parameters[0];
-	Color color(com.parameters[1] << 3,
-				com.parameters[2] << 3,
-				com.parameters[3] << 3,
-				com.parameters[4] << 3);
+	Color color(com.parameters[1] * 255 / 31,
+		com.parameters[2] * 255 / 31,
+		com.parameters[3] * 255 / 31,
+		com.parameters[4] * 255 / 31);
```

Comment:
ding

Reply:
Hmm, this is another rule for our style guideline.
Though I personally like "*indent* with tabs, *align* with spaces" more.